### PR TITLE
PHP7 code improvements

### PR DESCRIPTION
**Pull request by ahilles107**

### CODE REVIEW

File: newscoop/template_engine/metaclasses/MetaSubscriptions.php [hunk 1]
```diff
@@ -25,32 +25,47 @@ public function __construct($publicationId, $userId)
         }
     }
 
-    public function has_section($sectionNumber) {
+    public function has_section($sectionNumber)
+    {
```
Comment (takeit) [inline newscoop/template_engine/metaclasses/MetaSubscriptions.php:29]: is this class used somewhere?

**Comment by takeit** [comment]
there is no travis job running for that PR

File: newscoop/template_engine/metaclasses/MetaSubscriptions.php (hunk 1 above)
Comment (ahilles107) [inline newscoop/template_engine/metaclasses/MetaSubscriptions.php:29]: Yes, this is only way to check if user have access to content. 

```
{{ $subscription = $gimme->user->subscriptions->has_publication($gimme->publication->identifier) }}
{{ if !$gimme->article->is_public && !$subscription }}
....
{{ /if }}  
```

**Comment by ahilles107** [comment]
no idea why ;/

**Comment by ahilles107** [comment]
it's here: https://travis-ci.org/ahilles107/Newscoop/builds/138017642

File: newscoop/classes/SaaS.php [hunk 2]
```diff
@@ -42,8 +42,8 @@ public static function singleton()
      */
     public function hasPermission($p_permissionString)
     {
-        if (array_key_exists('permissions', $this->saasConfig)) {
-            $permissions = $this->saasConfig['permissions'];
+        if (array_key_exists('permissions', self::$saasConfig)) {
+            $permissions = self::$saasConfig['permissions'];
```
Comment (takeit) [inline newscoop/classes/SaaS.php:46]: Check https://github.com/ahilles107/Newscoop/blob/e76c2548d225fc72181f8bc607b40e22ae33b33a/newscoop/classes/SaaS.php#L76 it is still not changed here.

Comment (ahilles107) [inline newscoop/classes/SaaS.php:46]: no idea how i missed it, will commit it in a minute.

**Pull request merged**